integration-api-client/src/main/java/org/symphonyoss/integration/api/client/SimpleHttpApiClient.java

```diff
@@ -86,7 +86,7 @@ public String escapeString(String str) {
         getInvocationBuilder(path, client, queryParams, headerParams);
 
     Response response = invocationBuilder.get();
-    return handleResponse(returnType, response);
```

Comment:
It is an anti-pattern to open and close a resource in two different methods.  Put try/finally in each of these methods around the both lines (invoking request and handling response).